packages/components/stories/check-list/check-list.stories.js

```diff
@@ -17,23 +16,19 @@ storiesOf('CheckList', module)
                 {
                     content: 'Item with an action',
                     status: 'action',
-                    onClick: () => console.log('clicked'),
```

Comment:
i'm thinking that since storybook is a place to showcase the component isn't it ok to have these consoles?

Reply:
Sure. I'll put it back